cmd/minishift/cmd/start.go

```diff
@@ -508,7 +512,7 @@ func initSubscriptionManagerFlags() *flag.FlagSet {
 	subscriptionManagerFlagSet := flag.NewFlagSet(commandName, flag.ContinueOnError)
 	subscriptionManagerFlagSet.String(configCmd.Username.Name, "", "Username for the virtual machine registration.")
 	subscriptionManagerFlagSet.String(configCmd.Password.Name, "", "Password for the virtual machine registration.")
-	subscriptionManagerFlagSet.BoolVar(&skipRegistration, configCmd.SkipRegistration.Name, false, "Skip the virtual machine registration.")
+	subscriptionManagerFlagSet.BoolVar(&cmdUtil.SkipRegistration, configCmd.SkipRegistration.Name, false, "Skip the virtual machine registration.")
```

Comment:
this should be in a registration package, such as `cmdRegistration`, not `cmdUtil`.